notebooks/external_convergence_shear_distributions_tutorial.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>can you make a title and a description of what the notebook is about?</p><p> </p><p>Also note, there are different title sizes depending on how many '#' you are using.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>